src/test/resources/regressions/examples/infinite_list.gobra

```diff
@@ -9,7 +9,7 @@ type node struct {
 };
 
 pred infList(headPtr *node) {
-  headPtr != nil && acc(headPtr.value) && acc(headPtr.next) && infList(headPtr.next)
+  headPtr != nil && acc(&headPtr.value) && acc(&headPtr.next) && infList(headPtr.next)
```

Comment:
After seeing it in action, I agree that it makes more sense to expect an address in `acc(...)`

Reply:
That is reassuring to hear.